cmd/telemetrygen/internal/logs/worker.go

```diff
@@ -35,7 +56,9 @@ func (w worker) simulateLogs(res *resource.Resource, exporter exporter) {
 		nRes := logs.ResourceLogs().AppendEmpty().Resource()
 		attrs := res.Attributes()
 		for _, attr := range attrs {
-			nRes.Attributes().PutStr(string(attr.Key), attr.Value.AsString())
```

Comment:
Why does this only apply to logs?

Reply:
For logs it was easy to implement, since they are constructed by the telemetrygen. Traces are using sdk - https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/cmd/telemetrygen/internal/traces/traces.go#L87.

Since I didn't needed traces and I wasn't sure whether it's good idea at all, I didn't wanted to invest more effort into implementing traces.